pkg/sypgp/sypgp.go

```diff
@@ -258,6 +258,22 @@ func LoadPrivKeyring() (openpgp.EntityList, error) {
 	return el, nil
 }
 
+// LoadPubKeyringFromFile loads the public key from a local file given from a path into an EntityList
+func LoadPubKeyringFromFile(path string) (openpgp.EntityList, error) {
+
+	f, err := os.Open(path)
+	if err != nil {
+		return nil, err
+	}
+
+	el, err := openpgp.ReadKeyRing(f)
+	if err != nil {
+		return nil, err
+	}
+
+	return el, nil
+}
+
 // LoadPubKeyring loads the public keys from local store into an EntityList
 func LoadPubKeyring() (openpgp.EntityList, error) {
```

Comment:
It's probably a good idea that you change the existing `LoadPubKeyring` to call `LoadPubKeyringFromFile(PublicPath())`